internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -4388,6 +4551,561 @@ func TestHandleVerifyChain(t *testing.T) {
 	}})
 }
 
+func TestHandleGetWork(t *testing.T) {
+	t.Parallel()
+
+	blockStr := "070000000380b2db1300aabd5b1fb25821b0a3acb6773c5005b9b504" +
```

Comment:
Any reason you're not just using the existing `block432100` here?

Reply:
No particular reason, the first thought was to get a mined simnet block, submission and work for the test. I realize it'd be less code to use `block432100`. Rectifying.